web/src/main/kotlin/no/nav/su/se/bakover/web/routes/søknadsbehandling/SøknadsbehandlingRoutes.kt

```diff
@@ -73,6 +73,15 @@ internal fun Route.søknadsbehandlingRoutes(
     data class OpprettBehandlingBody(val soknadId: String)
     data class WithFritekstBody(val fritekst: String)
 
+    val feilVedHentingAvSaksbehandlerEllerAttestant = InternalServerError.errorJson(
```

Comment:
Antar at disse 2 også gjelder for revurdering. Er det lett å gjenbruke de for både søknadsbehandling og revurdering?

Reply:
Fra det jeg sjekket, så er disse bare i søknadsbehandling. med mindre det ligger noen som jeg ikke har sett